package.json

```diff
@@ -28,7 +28,7 @@
   "config": {
     "npmPublishTagProduction": "latest",
     "npmPublishTagTesting": "beta",
-    "nodeVersion": "8"
+    "nodeVersion": "12"
   },
```

Comment:
I think this config object isn't actually used anymore

Reply:
Yea don't think so either, I can remove and add the `"engines"` field if we want that